Dockerfile

```diff
@@ -82,3 +82,5 @@ RUN xmlcatalog --noout --add "delegatePublic" \
         "http://www.oasis-open.org/docbook/" \
         "file:///etc/xml/docbook" \
         /etc/xml/catalog
+
+WORKDIR "/srv/asciidoc"
```

Comment:
Maybe set your editor setting to "Ensure new line at file end" :)

Reply:
I honestly don't know why I didn't think to do that ever.